homeassistant/components/kef/media_player.py

```diff
@@ -218,6 +222,14 @@ def supported_features(self):
         if self._supports_on:
             support_kef |= SUPPORT_TURN_ON
 
+        if self._source in ("Wifi", "Bluetooth"):
```

Comment:
Supported features are not allowed to change during the entity lifetime.

See https://github.com/home-assistant/architecture/issues/1 and https://github.com/home-assistant/architecture/issues/338.

Reply:
I removed that.

It seemed to work fine though, also something that would be great to have in the future.